src/features/user/Account/Recurrency.tsx

```diff
@@ -26,45 +21,38 @@ export default function Recurrency({
   recurrencies,
   fetchRecurrentDonations,
 }: Props): ReactElement {
-  const { t, i18n } = useTranslation(['me']);
+  const { t } = useTranslation(['me']);
   const [selectedRecord, setSelectedRecord] = React.useState<number | null>(0);
-  const [openModal, setOpenModal] = React.useState(false);
+  const [isModalOpen, setIsModalOpen] = React.useState(false);
+  // const [openModal, setOpenModal] = React.useState(false);
```

Comment:
if this state not needed anymore then remove it

Reply:
Done